bwa/src/game.cpp

```diff
@@ -1,32 +1,47 @@
+#include <exception>
 #include <string>
 #include <utility>
 #include "game.hpp"
 
 constexpr const char* WINDOW_TITLE = "Bubble Warrior Adventures!";
 
 bwa::Game::Game() {
+	// Loads the config into the sol::state
 	_lua.script_file("config.lua");
+	// Converts the config's x and y resolution into a table
 	sol::table resolution = _lua["resolution"];
-	_font.loadFromFile(_lua["game_fonts"]["normal"]);
+	// Checks for font file and sets _font to it
+	if (!_font.loadFromFile(_lua["game_fonts"]["normal"]))
+	{
```

Comment:
Remove open curly brace since if statement only contains one statement inside of it so it can be omitted